windows/win_copy.py

```diff
@@ -48,16 +48,22 @@
 
 EXAMPLES = '''
 # Copy a single file
-- win_copy: src=/srv/myfiles/foo.conf dest=c:\\TEMP\\foo.conf
+- win_copy:
+    src: /srv/myfiles/foo.conf
+    dest: c:\TEMP\foo.conf
 
 # Copy the contents of files/temp_files dir into c:\temp\.  Includes any sub dirs under files/temp_files
-# Note the use of unix style path in the dest.  
+# Note the use of unix style path in the dest.
 # This is necessary because \ is yaml escape sequence
-- win_copy: src=files/temp_files/ dest=c:/temp/
+- win_copy:
+    src: files/temp_files/
+    dest: c:/temp/
```

Comment:
`/` or `\` for `dest:` on Windows machines, we have both listed in this module

The note on line 56 isn't clear